react/src/components/common/SnackbarWrapper.tsx

```diff
@@ -25,7 +25,7 @@ export default function SnackbarWrapper({children}: SnackbarWrapperProps): JSX.E
   return (
     <>
       {children}
-      <Toast severity={responseState?.severity} show={showToast} message={responseState?.message} onClose={(): void => setShowToast(false)} />
+      <Toast severity={responseState?.severity} show={showToast} message={responseState?.message} onClose={(): void => {setShowToast(false); responseState?.callback?.()}} />
```

Comment:
refactor 'onClose={(): void => {setShowToast(false); responseState?.callback?.()}}' to a handleClose function.